src/main/flight/pid.c

```diff
@@ -445,8 +445,12 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     horizonFactorRatio = (100 - pidProfile->horizon_tilt_effect) * 0.01f;
     maxVelocity[FD_ROLL] = maxVelocity[FD_PITCH] = pidProfile->rateAccelLimit * 100 * dT;
     maxVelocity[FD_YAW] = pidProfile->yawRateAccelLimit * 100 * dT;
-    const float ITermWindupPoint = (float)pidProfile->itermWindupPointPercent / 100.0f;
-    ITermWindupPointInv = 1.0f / (1.0f - ITermWindupPoint);
+    const float ITermWindupPoint = pidProfile->itermWindupPointPercent / 100.0f;
+    if (ITermWindupPoint == 1.0f) {
+        ITermWindupPointInv = 1.0f;
```

Comment:
This is actually unwanted behaviour. This means that iterm will always be lowered! This number is meant to be much higher always.

I think you misinterpreted this line further in the code:
dynCi *= constrainf((1.0f - motorMixRange) * ITermWindupPointInv, 0.0f, 1.0f);

value of 1 will always result in reduction of iterm!

Reply:
The idea of dynCi is to reduce iTerm only.  But yes I made a mistake in line 990:
`if ((axis <= FD_PITCH) && (ITermWindupPointInv < 1.0f)) {`
it should be 
`if ((axis <= FD_PITCH) && (ITermWindupPoint < 1.0f)) {`